doc/content/devices/best-practices/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Best Practices"
+description: ""
+weight:
+---
+
+This section contains best practices for building connected devices which use {{% tts %}} as a Network Server.
+
+<!--more-->
+
+LoRaWAN devices should always comply to the [LoRaWAN specification](https://www.lora-alliance.org/lorawan-for-developers), with special regard to minimizing unnecessary join requests and duty cycle limitations.
+
+The goal of these best practices is to optimize individual devices (especially for battery consumption) and the network as a whole (to reliably and efficiently serve more devices with the same number of gateways).
+
+## Unnecessary Join Requests
+
+The LoRaWAN specification warns especially against systematic rejoin in case of network failure. A device should keep the result of an activation in permanent storage if the device is expected to be turned off during its lifetime.
```

Comment:
```suggestion
The LoRaWAN specification advises specifically against periodic rejoin in case of network failure. A device should keep the result of an activation in permanent storage if the device is expected to be power-cycled during its lifetime.
```